### ISSUE TITLE
WoTE with only imitation Reward

### ISSUE THREAD

**Issue by TabGuigui** [liyingyanUCAS/WoTE#21]
Thanks for your excellent work.

In your ablation studies, you point out that 85.6 PDMS is only trained with 20epoch and in Table.4 you point out WoTE with only imitation reward is 83.5.

However, I use your original training set, which uses 30 epochs (I think it corresponds to 88.3), and only train with imitation reward loss and L1 refine loss, the PDMS is 82.5, which is far from the result in Table.4 where 20 epochs will get an 83.5 PDMS result with only imitation reward loss.

I wonder if there is something wrong I am doing to reproduce the ablation result?

Looking for your reply.
